learning-ojs/en/editorial-workflow.md

```diff
@@ -561,6 +561,10 @@ This will open a new window.
 
 ![](./assets/learning-ojs-3-ed-accept.png)
 
+Note that if the journal has enabled an article processing charge (APC) to be charged to authors, the option will appear at this stage to notify the author that the payment is due. Selecting "Request publication fee" will prompt a payment notification email to be sent to the author with payment instructions. For information about enabling author fees, please see the [Subscriptions chapter > Payment Types](./subscriptions) and [Distribution settings > Enable Payment](./settings-distribution).
```

Comment:
@kshuttle If you want to link to a specific section on a docs page, you can add an anchor link like this:

`[Subscriptions chapter > Payment Types](./subscriptions#payment-types)`
`[Distribution settings > Enable Payment](./settings-distribution#enable-payments)`

The anchor link is based on the heading text, but if you're ever not sure, you can check by clicking the "#" beside a heading on a docs page.